specification/resourcegraph/resource-manager/Microsoft.ResourceGraph/preview/2018-09-01-preview/graphquery.json

```diff
@@ -0,0 +1,535 @@
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Metadata describing a graph query for an Azure resource.",
+          "$ref": "#/definitions/GraphQueryPropertiesUpdateParameters"
+        }
+      },
+      "description": "The parameters that can be provided when updating workbook properties properties."
+    },
+    "GraphQueryPropertiesUpdateParameters": {
+      "description": "Properties that contain a workbook for PATCH operation.",
+      "properties": {
+        "description": {
+          "type": "string",
+          "description": "The description of a graph query."
+        },
+        "query": {
+          "type": "string",
+          "description": "KQL query that will be graph."
+        },
+        "resultKind": {
+          "description": "Enum indicating a type of graph query.",
+          "default": "basic",
+          "type": "string",
+          "enum": [
+            "basic",
+            "extended"
+          ],
+          "x-ms-enum": {
+            "name": "ResultKind",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "ErrorFieldContract": {
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "Property level error code."
+        },
+        "message": {
+          "type": "string",
+          "description": "Human-readable representation of property-level error."
+        },
+        "target": {
+          "type": "string",
+          "description": "Property name."
+        }
+      },
+      "description": "Error Field contract."
+    },
+    "GraphQueryError": {
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "Service-defined error code. This code serves as a sub-status for the HTTP error code specified in the response."
+        },
+        "message": {
+          "type": "string",
+          "description": "Human-readable representation of the error."
+        },
+        "details": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ErrorFieldContract"
+          },
+          "description": "The list of invalid fields send in request, in case of validation error."
+        }
+      },
+      "description": "Error message body that will indicate why the operation failed."
+    },
+    "OperationListResult": {
+      "description": "Result of the request to list Resource Graph operations. It contains a list of operations and a URL link to get the next set of results.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Operation"
+          },
+          "description": "List of Resource Graph operations supported by the Resource Graph resource provider."
+        }
+      }
+    },
+    "Operation": {
```

Comment:
>Operation [](start = 5, length = 9)

"isDataAction" is missing

IsDataAction:  Required. Indicates whether the operation applies to data-plane. Set the value to "true" for data-plane operations and "false" for ARM/control-plane operations.

Reply:
Not sure why we need this field. Could you elaborate more?